platform/cli/__tests__/captureTerminal.ts

```diff
@@ -1,72 +1,47 @@
-import { clone } from 'lodash';
-
 export function captureTerminal<T extends string[]>(
   fn: (argv: T) => any,
   argv: T
 ) {
-  const _exit = process.exit;
-  const _emit = process.emit;
+  let mockProcessExit = jest.spyOn(global.process, 'exit')
```

Comment:
Well, I wouldn't touch this file at all but the problem is that in node 8 typings `process.exit` has `never` return type that is incompatible with `void` so it's hard to simulate in tests (it should never return).